assets/sass/main.scss

```diff
@@ -1,5 +1,2 @@
-body {
-  width: 100%;
-  height: 100vh;
-  background-color: blueviolet;
-}
\ No newline at end of file
+@import './reseter';
+@import './constants';
```

Comment:
```suggestion
@import './constants';

```

Adding EOL (End Of Line) to improve this warning:

<img width="138" alt="image" src="https://user-images.githubusercontent.com/26731448/179877678-1ede9e89-cdb0-48ab-ad4e-fe47f93d77a2.png">